docs/src/inputs/prompt/prompt.jsx

```diff
@@ -1,132 +1,110 @@
-            description: 'Give an inline Prompt\'s action message a custom background color.',
-            allowedTypes: 'alert, success, warning',
-        }, {
-            name: 'inlineHorizontalAlign',
-            type: 'enum',
-            default: 'left',
-            description: 'An inline Prompt can be horizontal aligned to the left or the right.',
-            allowedTypes: 'left, right',
-        }, {
-            name: 'message',
-            type: 'string',
-            default: '',
-            description: 'Supply a Prompt a custom message.',
-            allowedTypes: '',
-        }, {
-            name: 'onClick',
-            type: 'func',
-            default: '',
-            description: 'Prompts can handle an onClick event.',
-            allowedTypes: '',
-        }, {
-            name: 'onNoClick',
-            type: 'func',
-            default: '',
-            description: 'An onClick event handler for the no button.',
-            allowedTypes: '',
-        }, {
-            name: 'onYesClick',
-            type: 'func',
-            default: '',
-            description: 'An onClick event handler for the yes button.',
-            allowedTypes: '',
-        }, {
-            name: 'style',
-            type: 'object',
-            default: '',
-            description: 'Supply any inline styles to the Prompt\'s container. Mainly used for padding and margins.',
-            allowedTypes: '',
+function DocsPrompt(props) {
+    const {
+        location: {
+            pathname,
         },
-    ];
+    } = props;
+
+    const {
+        description,
+        displayName,
+    } = rootDoc;
 
     return (
-        <Main page="headers">
+        <Main page={camelCase(displayName)}>
             <Main.Content>
-                <Card>
-                    <Header size="large">Props</Header>
+                <MarkdownContainer>
+                    <Typography
+                        className="description"
+                        variant="body1"
+                    >
+                        {description}
+                    </Typography>
 
-                    <TableProps props={tableProps} />
-                </Card>
+                    <Heading
+                        variant="h2"
+                    >
+                        Button With Confirming Prompt
+                    </Heading>
 
-                {/* Inline */}
-                <Header anchor="inline" size="large" style={{ marginTop: '55px' }} sub>
-                    Inline Prompt
-                    <Header.Subheader>
-                        A Button, Dropdown, or Link can have prompt attached to it asking for a confirmation from the end-user.
-                    </Header.Subheader>
-                </Header>
+                    <Example
+                        rawCode={require('!!raw-loader!./examples/exampleSuccessPrompt').default}
+                    >
+                        <ExampleSuccessPrompt />
+                    </Example>
```

Comment:
The JSX should be like...

```
<Main page={camelCase(displayName)}>
    <Main.Content>
        <MarkdownContainer>
            <Typography
                className="description"
                variant="body1"
            >
                {description}
            </Typography>
            
            <Heading
                anchorLink="solid-button"
                variant="h2"
            >
                Button with Alert Prompt
            </Heading>

            <Example
                rawCode={require('!!raw-loader!./examples/exampleFoo').default}
            >
                <ExampleFoo />
            </Example>
            
            <Heading
                anchorLink="solid-button"
                variant="h2"
            >
                Button with Warning Prompt
            </Heading>

            <Example
                rawCode={require('!!raw-loader!./examples/exampleBar').default}
            >
                <ExampleBar />
            </Example>
            
            <Heading
                anchorLink="solid-button"
                variant="h2"
            >
                Button with Informative Prompt
            </Heading>

            <Example
                rawCode={require('!!raw-loader!./examples/exampleBaz').default}
            >
                <ExampleBaz />
            </Example>
            
            <Heading
                anchorLink="solid-button"
                variant="h2"
            >
                Button with Confiming Prompt
            </Heading>

            <Example
                rawCode={require('!!raw-loader!./examples/exampleQux').default}
            >
                <ExampleQux />
            </Example>
        </MarkdownContainer>
        
        <ComponentVersionIdentifier
            pathname={pathname}
        />
    <Main.Content>
</Main>
```

Reply:
Resembling the mockup linked below:

https://www.sketch.com/s/6c8808fa-bf58-42e5-a0be-7e9d5798fc5f/a/VRRPro#Inspector

Reply:
Fixed